src/commands/command-fns/__tests__/blog.js

```diff
@@ -0,0 +1,92 @@
+const {rest} = require('msw')
+const blog = require('../blog')
+const {server} = require('../../../../tests/server')
```

Comment:
If you update `kcd-scripts` you should be able to change this to:

```suggestion
const {server} = require('server')
```

Because I just added `tests` to the modulesDirectories: https://github.com/kentcdodds/kcd-scripts/commit/4961e9741460825131f5de47d93d41b0a9ffe99d

Reply:
I'll be updating kcd-scripts soon and I'll push that so you can rebase.

Reply:
Ok, updated. You can rebase. Also, I added some utils you *might* find useful. They're still a tiny bit clunky though...

Reply:
the fake client will be very useful, nice 👍